.golangci.yml

```diff
@@ -40,10 +39,13 @@ issues:
     - text: "ST1016:"
       linters:
         - stylecheck
-    - path: "migrations"
-      text: "SA1019:"
+    - text: "SA1019:"
```

Comment:
This was popping for a lot of the IAVL changes, I think b/c we are using a wrapper, so I silenced them here by removing the "migrations" path. Might be a little overkill but I was spending too much time fixing this linter